chess_library.py

```diff
@@ -1,18 +1,16 @@
-from code import chess_move
-from code import chess_position
+from chess_move import ChessMove
+from chess_position import ChessPosition
 
-class ChessLibrary:
+def handoff(self, move):
+    #Give a ChessMove object to the third-party library
+    return 0
 
-    def handoff(self, move):
-        #use whatever method 3rd party library to hand it information from the object "move", which
-        #is a ChessMove object
-        return 0;
+def getmove(self):
+    #Asking Chess Library for a move, and storing it into a ChessPosition object
+    #In this example, the original pos is 0,0 and the final is 1,1
+    init = ChessPosition(0, 0)
```

Comment:
Look at the `ChessPosition.__init__` method, it asks for a column (a - h) and then a row (1-8). Update your code to give it a letter and then a number